command.go

```diff
@@ -3834,10 +3839,15 @@ func (cmd *baseCommand) executeAt(ifc command, policy *BasePolicy, deadline time
 			if deviceOverloadError(err) {
 				cmd.node.incrErrorCount()
 			}
+			// try to salvage the connection
+			if cmd.conn.salvageConnection && policy.TimeoutDelay > 0 {
```

Comment:
No reason to check for salvage connection since it is market by `policy.TimeoutDelay anyway`.